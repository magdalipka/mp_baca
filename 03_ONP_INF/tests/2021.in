21
INF: !(a+b)
INF: ~!(a+b)
INF: a+!b
ONP: a b + !
ONP: a b + ! ~
ONP: a b ! +
INF: !a*b+~(a*b)
ONP: a ! b * a b * ~ +
ONP: ta~xb~<<=
INF: x=~~a+b*c
ONP: xabc**=
ONP: ab+a~a-+
INF: a+b+(~a-a)
INF: x=~~a+b*c
INF: t=~a<x<~b
INF: ( a,+ b)/..[c3
ONP: ( a,b,.).c;-,*
ONP: abc++def++g+++
INF: x=a=b=c
ONP: xabc=== 
ONP: ab~+~
