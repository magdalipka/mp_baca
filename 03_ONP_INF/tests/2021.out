ONP: a b + !
ONP: a b + ! ~
ONP: a b ! +
INF: ! ( a + b )
INF: ~ ! ( a + b )
INF: a + ! b
ONP: a ! b * a b * ~ +
INF: ! a * b + ~ ( a * b )
INF: t = ~ a < ( x < ~ b )
ONP: x a ~ ~ b c * + =
INF: x = a * ( b * c )
INF: a + b + ( ~ a - a )
ONP: a b + a ~ a - +
ONP: x a ~ ~ b c * + =
ONP: t a ~ x < b ~ < =
ONP: a b + c /
INF: a * ( b - c )
INF: error
ONP: x a b c = = =
INF: x = a = b = c
INF: ~ ( a + ~ b )
